GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -3330,6 +3330,71 @@ internal TestAccessor(FormCommit formCommit)
                 string commitMessageText, bool usingCommitTemplate, bool ensureCommitMessageSecondLineEmpty)
                     => FormCommit.FormatCommitMessageFromTextBox(commitMessageText, usingCommitTemplate, ensureCommitMessageSecondLineEmpty);
         }
+
+        private void toWindowsCRLFToolStripMenuItem_Click(object sender, EventArgs e)
+        {
+            ConvertEndOfLines(Environment.NewLine);
```

Comment:
I'd be explicit here. Perhaps GE will return to Linux.
```suggestion
            ConvertEndOfLines("\r\n");
```